readme.md

```diff
@@ -196,6 +196,7 @@ On a default setup, running the command `kldload pty` should do the trick. If yo
 
 * **Bash**
 	* [sapegin/dotfiles](https://github.com/sapegin/dotfiles)’s [prompt](https://github.com/sapegin/dotfiles/blob/dd063f9c30de7d2234e8accdb5272a5cc0a3388b/includes/bash_prompt.bash) and [color theme](https://github.com/sapegin/dotfiles/tree/master/color) for `Terminal.app`.
+	* [anirudhb/purer](https://github.com/anirudhb/purer): Synchronous Git, NPM, and SSH check, shows outdated dependencies, able to resolve to relative path, and username is always shown. With color!
```

Comment:
NPM => npm

-

> able to resolve to relative path

What does this even mean?

> With color!

I don't see how this is a selling point? All prompts have color.